src/components/Preview.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Preview from './Preview';
+import { cleanup } from '@testing-library/react';
+
+afterEach(cleanup);
+
+describe('<Preview/>', () => {
+  it('renders without crashing', () => {
+    global.renderWithRedux(<Preview />);
+  });
+
+  it('renders appropriate content', () => {
+    // render
+    const { getByTestId } = global.renderWithRedux(
+      <Preview
+        expanded={true}
+        previewIdx={0}
+        frames={{ 1: 'test' }}
+        frameIDs={[1]}
+      />
+    );
+    expect(getByTestId('Frame-container').firstChild.alt).toBe('current frame');
+    expect(
```

Comment:
hmm, this expect and the one above it are already handled by another test, right? do you need them here too?

Reply:
Done